src/clarity-angular/popover/dropdown/dropdown-item.ts

```diff
@@ -16,7 +16,11 @@ export class DropdownItem {
     @HostListener("click")
     onDropdownItemClick(): void {
         if (this.dropdown.isMenuClosable && !this.el.nativeElement.classList.contains("disabled")) {
-            this.ifOpenService.open = !this.ifOpenService.open;
```

Comment:
Each Dropdown instance gets its own `IfOpenService` right? Is `tempDropdown` needed here?